packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-self-reg/main/java/summaryofsteps.md

```diff
@@ -1,45 +1,45 @@
 The sequence of steps for self-service registration is described in the following three sequence diagrams:
 
-### Start new user registration with password factor
+### Start a new user registration with the password authenticator
 
-The following diagram illustrates the beginning of the registration process where the user initiates their sign-in and enters their password.
+The following diagram illustrates the beginning of the registration process where the user initiates the sign-in flow and enters their password.
```

Comment:
```suggestion
The following diagram illustrates the beginning of the registration process when the user initiates the sign-in flow and enters their password.
```

Reply:
I'm pretty sure it should be "when", but I confess I'm not 100% sure.

Reply:
when does sound better :-)